python/ray/tune/execution/checkpoint_manager.py

```diff
@@ -37,8 +36,8 @@ def __init__(
 
         if checkpoint_config.num_to_keep == 0:
             raise RuntimeError(
-                "If checkpointing is enabled, Ray Tune requires `keep_checkpoints_num` "
-                "to be None or a number greater than 0"
+                "Tune requires `num_to_keep` in `CheckpointConfig` to be None "
+                "or a number >= 1."
```

Comment:
```suggestion
                "Tune requires `num_to_keep` in `CheckpointConfig` to be None "
                "or an integer >= 1."
```

Reply:
I removed this error, since the updated config validation throws an error immediately already.